Source/multi.h

```diff
@@ -23,7 +23,7 @@ struct Player;
 struct GameData {
 	int32_t size;
 	/** Used to initialise the seed table for dungeon levels so players in multiplayer games generate the same layout */
-	uint32_t dwSeed;
+	uint32_t gameSeed[4];

```

Comment:
Hm this will move the version and programid which will cause issues for older versions trying to determin what client it is.

Reply:
I was thinking it would fail before getting to that point, because the size changed.

https://github.com/diasurgical/devilutionX/blob/ad452cb6c03fd0f8ec6c2c616fb10a7f46fb5b4b/Source/dvlnet/base_protocol.h#L364-L365

Reply:
I think that means that they won't see it rather then seeing that there are clients running other versions. Not critical, but not ideal. Might be worth updating that code, and make sure that we in the future only append to this structure.

Looks like gamelist will have a similar issue: https://github.com/diasurgical/devilutionx-gamelist/blob/main/main.cpp#L270